### PR TITLE
AAP18813 - Adding OAuth/Application content to the access management guide

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR moves and updates the OAuth and Applications content into the Access management guide.

### CODE REVIEW

File: downstream/modules/platform/proc-controller-create-application.adoc [hunk 1]
```diff
@@ -17,19 +18,26 @@ The *Create Application* page opens.
...
-
-* *Name* (required): give a name for the application you want to create
-* Optional: *Description*: give a short description for your application
-* *Organization* (required): give an organization with which this application is associated
-* *Authorization grant type* (required): select one of the grant types to use for the user to get tokens for this application.
-For more information, see link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/automation_controller_administration_guide/index#ref-controller-password-grant-type[Application using password grant type] section of the _{ControllerAG}_.
-* *Client Type* (required): select the level of security of the client device.
-* *Redirect URIS*: give a list of allowed URIs, separated by spaces.
++
+Name:: (required) Enter a name for the application you want to create
```
Comment (ariordan-redhat) [inline downstream/modules/platform/proc-controller-create-application.adoc:22]:
```suggestion
Name:: (required) Enter a name for the application you want to create.
```

File: downstream/modules/platform/proc-controller-create-application.adoc [hunk 2]
```diff
@@ -17,19 +18,26 @@ The *Create Application* page opens.
...
+Description:: (optional) Include a short description for your application
```
Comment (ariordan-redhat) [inline downstream/modules/platform/proc-controller-create-application.adoc:23]:
```suggestion
Description:: (optional) Include a short description for your application.
```

File: downstream/modules/platform/proc-controller-create-application.adoc [hunk 3]
```diff
@@ -17,19 +18,26 @@ The *Create Application* page opens.
...
+Organization:: (required) Select an organization with which this application is associated
```
Comment (ariordan-redhat) [inline downstream/modules/platform/proc-controller-create-application.adoc:24]:
```suggestion
Organization:: (required) Select an organization with which this application is associated.
```

File: downstream/modules/platform/ref-gw-access-rules-apps-tokens.adoc [hunk 4]
```diff
@@ -0,0 +1,24 @@
+[id="ref-gw-access-rules-apps-tokens"]
+
+= Access Rules for applications and tokens
+
+(ref-gw-access-rules-apps-tokens.adoc)
```
Comment (ariordan-redhat) [inline downstream/modules/platform/ref-gw-access-rules-apps-tokens.adoc:5]:
```suggestion
```

File: downstream/assemblies/platform/assembly-gw-token-based-authentication.adoc [hunk 5]
```diff
@@ -4,10 +4,26 @@ ifdef::context[:parent-context: {context}]
...
+= Configuring access to external applications with token-based authentication
 
-[TBD] The content for this is in progress in separate link:https://docs.google.com/document/d/1GLk5zcD-8hZxU61Ik8wASDofkX7iQo5IJvkCOv5gRMw/edit?usp=sharing[Applications] and link:https://docs.google.com/document/d/1w9KjK8CybZ3w0u0DGNf0rNDC72vomwbdt5rZkUZCaBk/edit#bookmark=id.1rtgxk9mwxi0[Token-based authentication] drafts. Once converted/updated in AsciiDoc and in the repo, update this section to include those modules. 
+Token-based authentication permits authentication of third-party tools and services with the platform through integrated OAuth 2 token support, and allows you to access external applications without having to store your password on disk.
 
+For more information on the OAuth2 specification, see link:https://docs.google.com/document/d/1mEYguv3V0WUUHBMc8tsgqlK8RB3ro8IZSlQmMuFfhww/edit#heading=h.x3j3jrhlprvm[The OAuth 2.0 Authorization Framework].
+
+For more information on using the `manage` utility to create tokens, see link:https://docs.google.com/document/d/1mEYguv3V0WUUHBMc8tsgqlK8RB3ro8IZSlQmMuFfhww/edit#heading=h.x3j3jrhlprvm[Token and session management].
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/assemblies/platform/assembly-gw-token-based-authentication.adoc:13]: Ooops, good catch!

File: downstream/modules/platform/proc-controller-apps-create-tokens.adoc [hunk 6]
```diff
@@ -2,49 +2,51 @@
...
-This opens a separate window that enables you to choose from the available options.
-Use the Search bar to filter by name if the list is extensive.
-Leave this field blank if you want to create a Personal Access Token (PAT) that is not linked to any application.
-* Optional: *Description*: give a short description for your token.
-* *Scope* (required): specify the level of access you want this token to have.
-
-. Click btn:[Create token], or click btn:[Cancel] to abandon your changes.
 +
-After you save the token, the newly created token for the user is displayed with the token information and when it expires.
+Application:: enter the name of the application with which you want to associate your token. Alternatively, you can search for it by clicking the Browse. This opens a separate window that enables you to choose from the available options. Select Name from the filter list to filter by name if the list is extensive.
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-apps-create-tokens.adoc:24]: Fixed.

File: downstream/modules/platform/proc-controller-create-application.adoc (hunk 1 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-create-application.adoc:22]: Fixed.

File: downstream/modules/platform/proc-controller-create-application.adoc (hunk 2 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-create-application.adoc:23]: Fixed.

File: downstream/modules/platform/proc-controller-create-application.adoc (hunk 3 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-create-application.adoc:24]: Fixed.

File: downstream/modules/platform/ref-gw-access-rules-apps-tokens.adoc (hunk 4 above)
Comment (dcdacosta) [reply downstream/modules/platform/ref-gw-access-rules-apps-tokens.adoc:5]: Gremlins abound! Fixed.

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**